report_qweb_signer/models/report.py

```diff
@@ -145,7 +147,8 @@ def pdf_sign(self, pdf, certificate):
             raise Warning(
                 _('Signing report (PDF)'),
```

Comment:
But It's not what i see here: https://github.com/odoo/odoo/blob/8.0/openerp/exceptions.py#L33
isn't it?